src/notebooks/controllers/vscodeNotebookController.ts

```diff
@@ -309,7 +309,12 @@ export class VSCodeNotebookController implements Disposable, IVSCodeNotebookCont
     private updateDisplayData() {
         const displayData = this.displayDataProvider.getDisplayData(this.connection);
         this.controller.label = displayData.label;
-        this.controller.description = displayData.description;
+        if (this.featureManager.features.kernelPickerType === 'Insiders' && displayData.serverDisplayName) {
+            // MRU kernel picker doesn't show controller kind/category, so add server name to description
+            this.controller.description = displayData.description
+                ? `${displayData.description} (${displayData.serverDisplayName})`
```

Comment:
We can do this concatenation in the `ConnectionDisplayDataProvider` it self
The goal of the `ConnectionDisplayDataProvbider` is to have all of the logic for `deatils`, `description` and `lable` in one place
Else this same code could end up having to be copied to other places if we use quick pick or a controller.

Reply:
This code would not be necesary, i.e. no need to change any code in this file.